### PR TITLE
Certificates in Compass Director

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Setup certificates access to Compass Director

**Related issue(s)**
https://github.com/kyma-incubator/compass/issues/116

### CODE REVIEW

File: chart/compass/charts/connector/templates/certs-setup-job.yaml [hunk 1]
```diff
@@ -15,8 +15,6 @@ spec:
         sidecar.istio.io/inject: “false”
     spec:
       serviceAccountName: {{ .Chart.Name }}-certs-setup-job
-      securityContext:
```
Comment (pkosiec) [inline chart/compass/charts/connector/templates/certs-setup-job.yaml]: I believe that this was added on purpose by @mwieczorek

File: chart/compass/charts/gateway/templates/oathkeeper-rules.yaml [hunk 2]
```diff
@@ -46,4 +46,32 @@ spec:
...
   - handler: id_token
     config:
-      claims: {{ .Values.oathkeeper.idTokenConfig.claims | quote }}
\ No newline at end of file
+      claims: {{ .Values.oathkeeper.idTokenConfig.claims | quote }}
+---
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: compass-gateway-certs
```
Comment (pkosiec) [inline chart/compass/charts/gateway/templates/oathkeeper-rules.yaml]: please rename it to compass-director-certs

File: chart/compass/charts/gateway/templates/oathkeeper-rules.yaml [hunk 3]
```diff
@@ -46,4 +46,32 @@ spec:
...
+spec:
+  description: Configuration of oathkeeper for secure endpoint of compass gateway
+  upstream:
+    url: http://compass-gateway.{{ .Release.Namespace }}.svc.cluster.local:3000
+  match:
```
Comment (pkosiec) [inline chart/compass/charts/gateway/templates/oathkeeper-rules.yaml]: Could you please move there also a rule for Connector? If you don't want to merge these two rules into one for Gateway, then at least please keep it in one file. Thanks!

File: chart/compass/charts/connector/templates/certs-setup-job.yaml (hunk 1 above)
Comment (Szymongib) [reply chart/compass/charts/connector/templates/certs-setup-job.yaml]: It was defined in two places, look at the line 30

**Pull request merged**